packages/elements-core/src/components/Docs/Model/Model.spec.tsx

```diff
@@ -16,6 +18,10 @@ const exampleSchema: JSONSchema7 = {
     propA: {
       type: 'string',
       enum: ['valueA'],
+      // @ts-ignore
+      'x-enum-descriptions': {
```

Comment:
For some reason `JSONSchema7` don't support vendor extensions. It's probably a OpenAPI enhancement for JSON. schema?
I am not sure if that's expected but that is the reason for the `@ts-ignore`.